screens/RegisterScreen.tsx

```diff
@@ -61,15 +57,27 @@ export default function RegisterScreen({ navigation }) {
       />
       <View style={styles.bigMargin} />
       <TouchableOpacity
-        style={styles.button}
+        style={ButtonStyles.container}
         onPress={() => {
           register();
           setPassword("");
         }}
       >
-        <Text style={styles.button_text}>Register</Text>
+        <Text style={ButtonStyles.buttonText}>Register</Text>
       </TouchableOpacity>
 
+      <View style={{marginTop: 32, alignItems: 'center'}}>
+        <Text style={styles.body}>Already have an account?</Text>
+        <LinkButton
+        darkColor="#fff"
+        onPress={() => {
+          navigation.navigate("Login");
+        }}
+        title='Log in'
+      />
+
+      </View>
```

Comment:
There was no way to get back to the login screen

Reply:
true